scout/adapter/mongo/case.py

```diff
@@ -237,6 +238,30 @@ def cases(self, owner=None, collaborator=None, query=None, skip_assigned=False,
                     {'individuals.display_name': {'$regex': name_query}},
                 ]
 
+        if within_days:
+            verbs = []
+            if finished or has_causatives:
```

Comment:
finished means archived or solved, but it is possible to archive cases without solving them (without having a causative) , you should use the "archive" key in this case